Sources/TuistGenerator/Linter/TargetLinter.swift

```diff
@@ -73,14 +73,20 @@ class TargetLinter: TargetLinting {
     private func lintProductName(target: Target) -> [LintingIssue] {
         var allowed = CharacterSet(charactersIn: "abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789_")
 
-        if target.product == .app {
+        if target.product == .app || target.product == .commandLineTool {
```

Comment:
nit: I would define and assing a `let allowsDot = target.product == .app || target.product == .commandLineTool` variable here, and reuse it below

Reply:
Thanks for the comments!